docusaurus/docs/create-a-plugin/develop-a-plugin/build-a-panel-plugin.md

```diff
@@ -53,7 +53,7 @@ The [PanelProps](https://github.com/grafana/grafana/blob/747b546c260f9a448e2cb56
 
 You can access the panel properties through `props`, as seen in your plugin.
 
-**src/SimplePanel.tsx**
+**src/components/SimplePanel.tsx**
 
 ```js
 const { options, data, width, height } = props;
```

Comment:
further below is talk of yarn dev and I think we could give an example of changing the colour given its using a theme colour.